python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -156,9 +156,16 @@ def callback(self, pre, post, node_map):
         pad_width = pad.attrs.pad_width
         top_pad, bottom_pad = pad_width[-2]
         left_pad, right_pad = pad_width[-1]
+        
+        pad_mode = pad.attrs.pad_mode
+        
+        if pad_mode == "constant":
+            padding = [top_pad, left_pad, bottom_pad, right_pad]
 
-        padding = [top_pad, left_pad, bottom_pad, right_pad]
-
+        if pad_mode == "reflect":
```

Comment:
Can you add a single op test that will trigger this logic?

Reply:
sanity test to trigger this logic is present [here](https://github.com/tenstorrent/tt-forge-fe/pull/877/files#diff-9094f8f34c15528e4fad062984363b21b8e2fdd4b19c6d49878c16cebea33484R18-R48:~:text=verify(-,inputs,-%2C%20framework_model%2C))

Reply:
If it is, why xfail error message is: `reason="error: 'ttnn.conv2d' op Bias must only have data on the final dimenstion")` for all tests? Shouldn't it be something related to data mismatch?

Reply:
Error mentioned in XFAIL is not because of above fix. `'ttnn.conv2d' op Bias must only have data on the final dimenstion")` issue occurs in most of the CNN models and it yet to be resolved  -> https://github.com/tenstorrent/tt-forge-fe/issues/637